Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.m

```diff
@@ -148,12 +152,12 @@ - (BOOL)isDefaultVariant
 {
 	if ([self.variants count] == 1) {
 		BUYProductVariant *productVariant = [self.variants firstObject];
-		BUYOptionValue *optionValue = [productVariant.options anyObject];
-		NSString *defaultTitleString = @"Default Title";
-		NSString *defaultString = @"Default";
-		if ([productVariant.title isEqualToString:defaultTitleString] &&
-			([optionValue.value isEqualToString:defaultTitleString] || [optionValue.value isEqualToString:defaultString])) {
-			return YES;
+		if ([productVariant.options count] == 1) {
+			BUYOptionValue *optionValue = [productVariant.options anyObject];
```

Comment:
How would feel about pushing this functionality down into the child objects? It might mean we need another test or two, but I'm thinking: `-[BUYOptionValue isDefault]` and `-[BUYProductVariant isDefault]`. The former  checks the name/value of its properties. The latter checks the number of options and ensures the one is default. Then this method would check the number of variants and make sure *it* is default.

It's really not the responsibility of the product to know all this stuff about the variant and option internals.